addons/mail/static/src/components/rtc_layout_menu/rtc_layout_menu.scss

```diff
@@ -2,45 +2,16 @@
 // Layout
 // ------------------------------------------------------------------
 
-.o_RtcLayoutMenu {
-    display: flex;
-    flex-direction: column;
-    padding: map-get($spacers, 1);
-}
-
-.o_RtcLayoutMenu_inputText  {
-    @include text-truncate();
-}
-
-.o_RtcLayoutMenu_separator {
-    width: 100%;
-}
-
-.o_RtcLayoutMenu_radio {
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-    font-size: 1.3em;
-}
-
 .o_RtcLayoutMenu_radioIcon {
     width: 5rem;
     height: 5rem;
-    margin-inline-end: map-get($spacers, 1);
-}
-
-.o_RtcLayoutMenu_radioInput {
-    margin: map-get($spacers, 3);
 }
 
 // ------------------------------------------------------------------
 // Style
 // ------------------------------------------------------------------
 
 .o_RtcLayoutMenu_radio {
-    cursor: pointer;
-    border-radius: 5px;
-
     &:hover {
         background-color: gray('100');
```

Comment:
Not a big fun of these custom fields design :/

For the scope of this task we could use default bootstrap components, https://getbootstrap.com/docs/4.3/components/forms/#radios.

If the visual result changes too much compared  to master, then let's use a combination of utility classes (bg-X, opacity-X-hover..)